### ISSUE TITLE
Support  `--colour` (Commonwealth English) 

### ISSUE THREAD

**Issue by panoply** [pnpm/pnpm#2603]
### pnpm version: 5.1.2

The spelling of "color" is American English whereas "colour" is British / Australian English (otherwise known as Commonwealth English).  Not really an issue per-say but would be nice if `--colour` could be passed and appropriated.

### Code to reproduce the issue:

1. `pnpm run <cmd> -r --colour`

### Actual behavior:

```cli
 ERROR  Unknown option: 'colour'
```

**Comment by zkochan** [comment]
I don't think it is worth adding but we can print a different error message. Something like

```
ERROR  Unknown option: 'colour'. Did you mean 'color'?
```

**Comment by panoply** [comment]
Sounds good.
